aspnetcore/migration/1x-to-2x/index.md

```diff
@@ -98,6 +98,20 @@ The adoption of this new 2.0 pattern is highly recommended and is required for p
 ```
 Unable to create an object of type '<Context>'. Add an implementation of 'IDesignTimeDbContextFactory<Context>' to the project, or see https://go.microsoft.com/fwlink/?linkid=851728 for additional patterns supported at design time.
 ```
+## Adding Additional Configurations
+
+In 2.0 projects, the boilerplate configuration code is done behind the scenes. By default, Environment Variables and Application Settings are loaded into startup so all that is required in *Startup.cs* is initializing `IConfiguration` with the injected instance. To add additional providers, or change the existing providers, utilize `ConfigureAppConfiguration` on `IWebHostBuilder` in *Program.cs*. The setup is similar to Core 1.x
```

Comment:
~~In 2.0 projects, the boilerplate configuration code is done behind the scenes.~~
In 2.0 projects, the boilerplate configuration code runs behind-the-scenes.

- Environment Variables --> environment variables
- Application Settings --> app settings
- Core 1.x --> 1.x: